Blocks/Persistence/test.cc

```diff
@@ -294,6 +299,12 @@ TEST(PersistenceLayer, File) {
   }
 
   EXPECT_EQ(
+      "#signature\t{\"exposed_namespace\":\"namespace\",\"top_level_name\":\"top_level_name\",\"schema\":{\"types\":[["
```

Comment:
Yes, but in that way it will be possible to overlook some "harmful" `StreamSignature` modification, and existing data files would become invalid. May be it's not so important..